pykeybasebot/kvstore_client.py

```diff
@@ -98,10 +98,10 @@ async def list_entrykeys(
         return keybase1.KVListEntryResult.from_dict(res)
 
     def is_deleted(self, res: keybase1.KVGetResult) -> bool:
-        return res.revision > 0 and res.entry_value == ""
+        return res.revision > 0 and res.entry_value is None
```

Comment:
i noticed the json bot lib PR keeps this backwards compatible by also checking whether res.entry_value  == "". should we be consistent across the bot libs?

Reply:
yup. fixed.